### ISSUE TITLE
[BUG] [Formatter] Incorrect indentation with template tags which do not have a {% endtag %}

### ISSUE THREAD

**Issue by jmichalicek** [djlint/djLint#1161]
- [ X] I'm on the [latest version](https://pypi.org/project/djlint/) of djLint
- [ X] I've searched the [issues](https://github.com/djlint/djLint/issues)
- [ X] I've read the [docs](https://djlint.com)

## System Info

- OS: Debian Bookworm
- Python Version (`python --version`): 3.12.2
- djLint Version (`djlint --version`): 1.36.4
- template language: e.g. mustache: Django

## Issue

Template tags do not format as expected if there is a template tag with no explicit ending tag;

```
{% if album.owner.profile_picture %}
  {% thumbnail "250x250" album.owner.profile_picture -- alt="Album owner profile image" class="rounded-full w-24 h-24" %}
  {% else %}
     <img src="{% static 'svg/icon-user-default.svg' %}"
               alt="user"
               class="rounded-full w-24 h-24">
  {% endif %}
```

Everything from `{% else %]` through `{% endif %}` should be dedented one level. This then continues to throw off the formatting throughout the rest of the html afterwards.

**Comment by jmichalicek** [comment]
While I saw it previously, and not technically the same in how we are reporting it, it looks like this is the same issue as #1113

**Comment by monosans** [comment]
This is a conflict between two third-party libraries rather than a djLint bug: djLint treats {% thumbnail %} as a block tag because sorl-thumbnail's {% thumbnail %}...{% endthumbnail %} is one, while easy-thumbnails uses a standalone {% thumbnail %} tag with the same name. djLint can't distinguish them by syntax alone. If you use the standalone variant, you can exclude it from block handling with the ignore_blocks setting. Closing, since either default breaks the other library's users.
